watertap/unit_models/zero_order/energy_recovery_zo.py

```diff
@@ -0,0 +1,40 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+This module contains a zero-order representation of an energy recovery unit
+operation.
+"""
+from pyomo.environ import Constraint, units as pyunits, Var
+from idaes.core import declare_process_block_class
+
+from watertap.core import build_pt, constant_intensity, ZeroOrderBaseData
+# Some more information about this module
+__author__ = "Marcus Holly"
+
+
+@declare_process_block_class("EnergyRecoveryZO")
+class EnergyRecoveryZOData(ZeroOrderBaseData):
+    """
+    Zero-Order model for an Energy Recovery unit operation.
+    """
+
+    CONFIG = ZeroOrderBaseData.CONFIG()
+
+    def build(self):
+        super().build()
+
+        self._tech_type = "energy_recovery"
+
+        build_pt(self)
+        constant_intensity(self)
+        self.electricity.setlb(None)
```

Comment:
This line is not needed. The bound on electricity is now set in the `constant_intensity` method,

Reply:
The bounds on `electricity` in the `constant_intensity` are set as` (0,None)`. However, the `electricity` in this unit is negative, I have to reset the lower bound to be `None`, otherwise the unit cannot be solved.

Reply:
This is going to create some other challenges then, as the costing framework assumes all flows are positive with a lower bound of 0 (and will raise a warning if this is not the case, which we don't want the users seeing).

Reply:
I've set the upper bound to be 0 for this case, I think it is reasonable at this point, the costing test passed.